lib/builder/builder.js

```diff
@@ -643,16 +645,10 @@ export default class Builder {
   }
 
   async unwatch() {
-    if (this.filesWatcher) {
-      this.filesWatcher.close()
-    }
-
-    if (this.customFilesWatcher) {
-      this.customFilesWatcher.close()
-    }
-
-    if (this.nuxtRestartWatcher) {
-      this.nuxtRestartWatcher.close()
+    for (const watcher in this.watchers) {
```

Comment:
I think this could be simplified. For example: `Object.values(this.watchers).forEach(watcher => watcher && watcher.close())`

Reply:
@manniL It's tempting, I know, but think about it. That'd be adding two unnecessary function calls to the stack. `Object.values()` and `forEach()`, in an obscurely constricted syntax. Using the native `for()` constructs makes it much more _cleaner_ IMHO :)

Reply:
for of is proper here

Reply:
I mean with a map